plugins/protocols/xdg-activation.cpp

```diff
@@ -0,0 +1,67 @@
+#include "wayfire/core.hpp"
+#include "wayfire/signal-definitions.hpp"
+#include "wayfire/view.hpp"
+#include <memory>
+#include <wayfire/plugin.hpp>
+#include <wayfire/view.hpp>
+#include <wayfire/nonstd/wlroots-full.hpp>
+#include <wayfire/window-manager.hpp>
+#include "config.h"
+
+extern "C"
+{
+#include <wlr/types/wlr_xdg_activation_v1.h>
```

Comment:
This should go to `wayfire/nonstd/wlroots-full.hpp`

Reply:
Added this include to `wayfire/nonstd/wlroots-full.hpp`